src/AbstractDay.php

```diff
@@ -147,6 +154,14 @@ protected function setDayOfWeek($dayOfWeek)
         $this->dayOfWeek = $dayOfWeek;
     }
 
+    /**
+     * Sets the opening intervals.
+     *
+     * @param array $openingIntervals
+     *
+     * @throws \InvalidArgumentException If no days are passed
+     * @throws \InvalidArgumentException If invalid interval is passed
```

Comment:
Can you merge this in one line please ?

`@throws \InvalidArgumentException If no days are passed or if invalid interval is passed`